118/fix links

JaeGif opened the pull request:
## This PR:

resolves #118

JaeGif committed:
fix recordsponge redirect [118]

JaeGif committed:
remove invalid discord channel invitations [118]

JaeGif committed:
removed redundant link to codepdx site [118]

Jared-Krajewski reviewed (changes requested):
Thanks for picking this issue up! To close #118 by removing icons, we would also need to fix the arrangement of the icons so they are more consistent. I think just centering the icons as seen in the last image would be fine. 

<img width="1551" alt="Screenshot 2024-04-15 at 9 20 19 AM" src="https://github.com/codeforpdx/codepdx_website/assets/114430929/1e87bd36-e866-4db5-85db-dbdd34e488c8">


<img width="395" alt="Screenshot 2024-04-15 at 9 31 09 AM" src="https://github.com/codeforpdx/codepdx_website/assets/114430929/10919e3e-e5ca-4b11-8890-8a338f66c0cd">

JaeGif committed:
add centering and spacing BP for links

JaeGif commented:
> Thanks for picking this issue up! To close #118 by removing icons, we would also need to fix the arrangement of the icons so they are more consistent. I think just centering the icons as seen in the last image would be fine.
> <img alt="Screenshot 2024-04-15 at 9 20 19 AM" width="1551" src="https://private-user-images.githubusercontent.com/114430929/322553771-1e87bd36-e866-4db5-85db-dbdd34e488c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxOTk2MDYsIm5iZiI6MTcxMzE5OTMwNiwicGF0aCI6Ii8xMTQ0MzA5MjkvMzIyNTUzNzcxLTFlODdiZDM2LWU4NjYtNGRiNS04NWRiLWRiZGQzNGU0ODhjOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxNVQxNjQxNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YmMwYWVjZmMzZDk3OTQ1Y2FiZTZhNTUyNGZmNWI0MTYyMmRhMTIzOTU1N2UxZGMzYjI2MWVjODMwMjRkY2IwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.kTax_QZkaZb8xelksvVgk14z5lucuCQ_f8rMwqERzVI"> <img alt="Screenshot 2024-04-15 at 9 31 09 AM" width="395" src="https://private-user-images.githubusercontent.com/114430929/322557845-10919e3e-e5ca-4b11-8890-8a338f66c0cd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxOTk2MDYsIm5iZiI6MTcxMzE5OTMwNiwicGF0aCI6Ii8xMTQ0MzA5MjkvMzIyNTU3ODQ1LTEwOTE5ZTNlLWU1Y2EtNGIxMS04ODkwLThhMzM4ZjY2YzBjZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxNVQxNjQxNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05OWYxZGM2NWI4MzkxZjhjZWQ2YTZhOWRkN2JiMGZmZTlmMzVjNGQ3OGEwZWQ3ODk5MmM5NTgxZDIxYjRlOWJlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4FOZ0avT9QPCRB0FSmoRSDV3POu69YIft0AZM2VO6e8">

Great idea. I changed the spacing to centered, and added some space using the breakpoints that the padding values have for consistency.

Jared-Krajewski reviewed (approved):
looks good. Thanks!
